templates/_layouts/default.html

```diff
@@ -112,12 +109,14 @@
     </div>
   </footer>
 
-  <script src="/static/js/modules/global-nav.js"></script>
-  <script src="/static/js/modules/cookie-policy.js"></script>
-  <script src="/static/js/tabs.js"></script>
+  <script src="{{ versioned_static('js/modules/global-nav.js') }}"></script>
+  <script src="{{ versioned_static('js/modules/cookie-policy.js')}}"></script>
+  <script src="{{ versioned_static('js/tabs.js') }}"></script>
   <script>
     cpNs.cookiePolicy();
-    canonicalGlobalNav.createNav({maxWidth:"72rem"});
+    if (typeof canonicalGlobalNav !== "undefined") {
+      canonicalGlobalNav.createNav({breakpoint: 940});
```

Comment:
```suggestion
      canonicalGlobalNav.createNav({breakpoint: 1035});
```
This needs to be the same as the navigation breakpoint threshold or there will be some visual glitches

Reply:
Thanks, updated.